modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -27,21 +27,62 @@ just like your other ownCloud files and folders.
 
 Compared to standard SMB access, WND has advanced features like:
 
-* User lockout prevention and password reset
-* More authentication mechanisms against the backend
-* Change notifications
-* Enhanced ACL support
+. User lockout prevention and password reset
+. More authentication mechanisms against the backend
+. Listen to change information triggered by the backend
+. Enhanced ACL support
+. Collaborative WND (CWND)
```

Comment:
There is also collaborative SMB, so I'm not sure if we want to mention the "CWND" as an advantage

Reply:
Yes, because it is specific to WND.
Even there is a similar functionality in SMB, SMB has eg no listener ect and there is the need for differenciation.
The upcoming CSMB description will then have only a subset of the CWND text